entity-framework/core/what-is-new/ef-core-10.0/breaking-changes.md

```diff
@@ -178,6 +182,137 @@ await context.Blogs.ExecuteUpdateAsync(s =>
 });
 ```
 
+<a name="compiled-model-private-methods"></a>
+
+### Compiled models now throw exception for value converters with private methods
+
+[Tracking Issue #35033](https://github.com/dotnet/efcore/issues/35033)
+
+#### Old behavior
+
+Previously, when using value converters with compiled models (using `dotnet ef dbcontext optimize`), EF would reference the converter type and everything worked correctly.
```

Comment:
@AndriySvyryd is this accurate? I haven't dived in, but the issue makes it sound like things didn't actually fully work, and this is just about throwing the right exception? If everything did work, does that mean we did some sort of breaking in change in order to support NativeAOT (which is experimental and for which we in principle shouldn't be breaking regular behavior)?